### PR TITLE
Change source of `download_url` from our own and fix `README`

### PR DESCRIPTION
**Pull request by rayrayraykk**
#18 #16 
As the title said.

### CODE REVIEW

**Review by xieyxclack — DISMISSED** [review]
Good job, it looks good to me and has resolved the proposed issue #18

**Comment by yxdyc** [comment]
LGTM.

File: federatedscope/core/auxiliaries/utils.py [hunk 1]
```diff
@@ -335,3 +337,31 @@ def merge_dict(dict1, dict2):
             else:
                 dict1[key].append(value)
     return dict1
+
+
+def download_url(url: str, folder='folder'):
```
Comment (DavdGao) [inline federatedscope/core/auxiliaries/utils.py:342]: Please check if there is other versions of the functions `download_url` and `extract`. (currently `mf` uses the torch version)

Comment (rayrayraykk) [reply federatedscope/core/auxiliaries/utils.py:342]: The minimal version `requirements-torch1.10.txt` contains `torchvision`,  so I think the usage of `download_url` from it is ok.

**Comment by rayrayraykk** [comment]
> LGTM. One minor question, is the `download_url` copied from `torch_geometric.data`? If yes, we may add simple comment to clarify this @rayrayraykk.

Thanks, I have added comments.

**Review by joneswong — APPROVED** [review]
approve

**Pull request merged**